koku/api/report/ocp_aws/ocp_aws_query_handler.py

```diff
@@ -39,6 +40,8 @@ def __init__(self, query_parameters, url_data,
             kwargs    (Dict): A dictionary for internal query alteration based on path
         """
         provider = 'OCP_AWS'
+        if kwargs.get('access'):
```

Comment:
Do we need AWS access controls here too? You can filter and group on AWS account.

Reply:
Call to super triggers the AWS access control call.

Reply:
👍